hdfs-scheduler/src/main/java/org/apache/mesos/hdfs/scheduler/HdfsNode.java

```diff
@@ -126,9 +118,9 @@ private ExecutorInfo createExecutor(String taskIdName, String nodeName, String e
               CommandInfo.URI
                 .newBuilder()
                 .setValue(
-                  String.format("http://%s:%d/%s", config.getFrameworkHostAddress(),
+                  String.format("http://%s:%d/%s?node=%s", config.getFrameworkHostAddress(),
```

Comment:
Is it necessary to host the nodeId? Can we instead get this information from SchedulerConf?